plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/EverytradeCsvMultiParser.java

```diff
@@ -32,46 +30,54 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
-public class EverytradeCsvParser implements ICsvParser {
+public class EverytradeCsvMultiParser implements ICsvParser {
     private static final String ID = EveryTradePlugin.ID + IPlugin.PLUGIN_PATH_SEPARATOR + "everytradeParser";
-    private static final Map<String, ExchangeDescriptor> exchangeDescriptors = new HashMap<>();
+    private static final Map<String, ExchangeParseDetail> exchangeDescriptors = new HashMap<>();
```

Comment:
UNDERSCORE_NAMING for `static final`s.
Also, don't use descriptors in the name.